pointerevents/pointerevent_pointerout_no_pointer_movement.html

```diff
@@ -44,19 +48,23 @@
   promise_test(async t => {
     await test_driver.click(target);
 
+    num_pointermoves = 0;
```

Comment:
Nit: I would also initialize `overlay.style.display` in each promise test so make them fully independent of each other.  E.g. if the first one fails for whatever reason before the style change, the second shouldn't fail automatically.